components/SwipeToDelete.tsx

```
@@ -0,0 +1,100 @@
+  const autoDeleteThreshold = containerWidth * 0.85;
+
+  const handleDelete = useCallback(() => {
+    swipeableRef.current?.close();
+    onDelete();
+  }, [onDelete]);
+
+  const renderRightActions = (
+    _progress: SharedValue<number>,
+    translation: SharedValue<number>,
+  ) => {
+    const animatedStyle = useAnimatedStyle(() => {
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)